conans/test/integration/basic_build_test.py

```diff
@@ -15,16 +15,18 @@ class BasicBuildTest(unittest.TestCase):
 
     def complete_build_flow_test(self):
         """In local user folder"""
-        files = cpp_hello_conan_files("Hello0", "0.1")
-        client = TestClient()
-        client.save(files)
-        command = "say_hello" if platform.system() == "Windows" else "./say_hello"
+        command = "bin\\say_hello" if platform.system() == "Windows" else "./bin/say_hello"
 
-        for install, lang, static in [("install", 0, True),
-                                      ("install -o language=1", 1, True),
-                                      ("install -o language=1 -o static=False", 1, False),
-                                      ("install -o static=False", 0, False)]:
-            client.run(install)
+        for lang, static in [(0, True), (1, True), (1, False), (0, False)]:
+            client = TestClient()
+            dll_export = False
+            if not static and client.default_compiler_visual_studio:
+                dll_export = True
+            files = cpp_hello_conan_files("Hello0", "0.1", language=lang,
```

Comment:
The behavior is also modified, it is different to create cpp_hello_files with lang, and static, that to pass options in the command line. This test is checking that options can be passed in the command line, **modifying** the defaults existing in the files.